src/utils/provideLocalStorageDataClass.ts

```diff
@@ -61,10 +69,38 @@ export function provideLocalStorageDataClass(className: string) {
     },
   })
 
+  async function initializeContent(
+    initialContent: { _id: string; [key: string]: unknown }[],
+  ) {
+    const initializedKey = `${recordKey}-initialized`
+    const initializedValue = `yes - already initialized with ${await sha1(
+      JSON.stringify(initialContent),
+    )}`
+
+    try {
+      if (localStorage.getItem(initializedKey) !== initializedValue) {
+        const initialRecord: Record<string, Record<string, unknown>> = {}
+        initialContent.forEach((item) => {
+          const id = item._id
+
+          if (Object.keys(initialRecord).includes(id)) {
+            throw new Error(`Duplicate _id ${id} in initialContent!`)
+          }
+
+          initialRecord[id] = item
+        })
```

Comment:
* [ ] This is a safety check for our own data. I don't see why we would need that. If we have an error in the initialContent we can just fix it.

Reply:
I would prefer to keep it - since it does not at much complexity to the code.

Reply:
After the latest refactoring this is the most complex (for example it uses exception driven control flow) but least useful (see above) part of the module.
* [ ] FYI If we really need it we could make it more readable by extracting a well-named helper method.